R/autoFormat.R

```diff
@@ -16,9 +16,15 @@
 #' @export
 autoFormat <- function(files = getFilesToLint(), ignoreLintFreeFiles = TRUE, lintAfterwards = TRUE) {
   if (ignoreLintFreeFiles) {
-    files <- files[sapply(files, function(aFile) length(lint(aFile)) > 0)]
+    # keep only files with linter warnings
+    files <- files[lapply(files, function(aFile) length(lint(aFile)) > 0)]
   }
-  style_file(files, strict = FALSE)
+
+  # strict = FALSE -> keep alignment spaces around assignments, keep extra newlines
+  # scope is set to not change indentation, otherwise the alignment of continuation lines would be messed up
+  style_file(files, strict = FALSE, scope = I(c("tokens", "line_breaks", "spaces")))
+  cat("Hint: In RStudio you can fix indentation in selected lines using ctrl + i.\n")
```

Comment:
```suggestion
  message("Hint: In RStudio you can fix indentation in selected lines using ctrl + i")
```
I would recommend using `message` instead of `cat` as messages can more easily be suppressed (via `suppressMessages`) if desired